src/WorkBC.Importers.Federal/Program.cs

```diff
@@ -33,12 +38,23 @@ private static async Task Main(string[] args)
             Parser.Default.ParseArguments<CommandLineOptions>(args)
                 .WithParsed(o => { options = o; })
                 .WithNotParsed(o => { Environment.Exit(0); });
+            
+            string connectionString = configuration.GetConnectionString("DefaultConnection");
+            var dbContext = new JobBoardContext(connectionString);
 
-            var service = new XmlImportService(configuration, options, logger);
+            //get API settings
+            var federalSettings = new FederalSettings();
+            configuration.GetSection("FederalSettings").Bind(federalSettings);
+            var baseUrl = federalSettings.FederalJobXmlRoot;
+            
+
+            var httpClient = CreateHttpClient(configuration);
```

Comment:
> The HttpClient object is intended to live for as long as your application needs to make HTTP requests. 
See:  https://web.archive.org/web/20180116003143/http://chimera.labs.oreilly.com:80/books/1234000001708/ch14.html

Reply:
Previously, we were instantiating an `HttpClient` every time a web request was made.